open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/rest/OMAGServerConfigsResponse.java

```diff
@@ -0,0 +1,130 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.adminservices.rest;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.odpi.openmetadata.adminservices.configuration.properties.OMAGServerConfig;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Set;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
+
+
+/**
+ * OMAGServerConfigsResponse is the response structure used on the OMAG REST API calls that returns a set of
+ * OMAGServerConfig objects as a response.
+ */
+@JsonAutoDetect(getterVisibility=PUBLIC_ONLY, setterVisibility=PUBLIC_ONLY, fieldVisibility=NONE)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class OMAGServerConfigsResponse extends AdminServicesAPIResponse
+{
+    private static final long    serialVersionUID = 1L;
+
+    private Set<OMAGServerConfig> serverConfigs = null;
+
+
+    /**
+     * Default constructor
+     */
+    public OMAGServerConfigsResponse()
+    {
+    }
+
+
+    /**
+     * Return the OMAGServerConfig objects.
+     *
+     * @return OMAGServerConfig objects
+     */
+    public Set<OMAGServerConfig> getOMAGServerConfigs()
+    {
+        if (serverConfigs == null)
```

Comment:
Need to test for empty set and return null.  The current logic is just returning the server configs irrespective of value